src/Config/MutationConfigObject.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Softonic\GraphQL\Config;
+
+use Softonic\GraphQL\Traits\JsonPathAccessor;
+
+class MutationConfigObject
```

Comment:
What's the difference between `MutationConfig` and `MutationConfigObject`?

Reply:
A `MutationConfig` contains different `MutationConfigObject`, accessable through the get() method. See `MutationConfigTest`

Reply:
`MutationVariableConfig`? At the end the mutation contains the variables of the mutation.

Reply:
Decided to name it MutationTypeConfig. Also created a MutationConfigBuilder.